templates/index.html

```diff
@@ -19,7 +19,7 @@ <h2>New item</h2>
             <input type="text" name="title" id="title-input" class="form-control" data-test-id="name-input">
           </div>
           <div class="col-auto">
-            <button type="submit" class="btn btn-success">Add Item</button>
+            <button type="submit" class="btn btn-success" {% if view_model.is_readonly %} disabled {%endif%}>Add Item</button>
```

Comment:
Rather than disabling this button without explanation, it's usually better UX to just hide this option entirely